web/src/components/dashboards/perses/dashboard-list.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { ReactNode, useMemo, type FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { DashboardLayout } from './dashboard-layout';
+import { useDashboardsData } from './hooks/useDashboardsData';
+
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Pagination,
+  Title,
+} from '@patternfly/react-core';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewFilters } from '@patternfly/react-data-view/dist/dynamic/DataViewFilters';
+import {
+  DataViewTable,
+  DataViewTh,
+  DataViewTr,
+} from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { DataViewTextFilter } from '@patternfly/react-data-view/dist/dynamic/DataViewTextFilter';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { useDataViewFilters, useDataViewSort } from '@patternfly/react-data-view';
+import { useDataViewPagination } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { ThProps } from '@patternfly/react-table';
+import { Link, useSearchParams } from 'react-router-dom-v5-compat';
+
+import { getDashboardUrl, usePerspective } from '../../hooks/usePerspective';
+import { Timestamp } from '@openshift-console/dynamic-plugin-sdk';
+const perPageOptions = [
+  { title: '10', value: 10 },
+  { title: '20', value: 20 },
+];
+
+interface DashboardName {
```

Comment:
It seems this interface name is misleading. Is it the dashboard row value for the name column?